content/news/releases/prestashop-9-alpha1-available.md

```diff
@@ -0,0 +1,147 @@
+
+–
+
+Developers working on PrestaShop solutions should pay close attention to the first two items. We also invite you to experiment with our new API.
+
+Although we are in the process of preparing documentation on the notable changes for the community, you can start by familiarizing yourself with the migration guides from PHP and release notes from Symfony:
+
+* [Migration from PHP 8.0 to PHP 8.1](https://www.php.net/manual/en/migration81.php)
+* [Migration from PHP 8.1 to PHP 8.2](https://www.php.net/manual/en/migration82.php)
+* [Migration from PHP 8.2 to PHP 8.3](https://www.php.net/manual/en/migration83.php)
+* [Release notes for Symfony 5.4](https://symfony.com/releases/5.4)
+* [Release notes for Symfony 6.4](https://symfony.com/releases/6.4)
+
+## How to test PrestaShop 9
+
+Click the button below to download the new version. The installation process remains the same as in previous versions. Please note: this version requires PHP 8.1 or higher.
+
+{{< cta "https://github.com/PrestaShop/PrestaShop/releases/tag/9.0.0-alpha1" >}}Download PrestaShop 9 Alpha 1 now!{{< /cta >}}
+
+If you plan to test the new API, it's important to look at the tutorial explaining how to use it properly. [This tutorial](#) will give you a clear idea of how to work with the API effectively, ensuring you can get the most out of your testing.
+
+## What’s next
+
+We are continuing our efforts to reach a stable version, with several key areas of development still in progress. Here is a brief outline of the crucial elements needed to achieve the release of the final version:
+
+Core:
+* Work on the new API continues to lay a great foundation for the future.
+
+Back office:
+* Symfony login page requires further improvements to be fully functional.
+
+Front office:
+* The decision regarding the Hummingbird theme and its presence in PrestaShop 9 has not yet been made.
+
+Documentation:
+* We will work to document all significant changes to the software to make it easier for developers to adapt their solutions to PrestaShop 9.
+* We will also prepare a guide on how to use the new API effectively and explain the basics of the new solution.
+
+Upgrade path:
+* You can use the "Local archive" method in the [autoupgrade](https://github.com/PrestaShop/autoupgrade) to try upgrading to PrestaShop 9 Alpha. There might be some issues, so please be cautious. We plan to test the upgrade path in the coming weeks.
+
+This list is not exhaustive, and we will continue to work on other areas of the software to ensure a stable release.
+
+## FAQ - last update on 15th of May, 2024
+
+**Can I upgrade to PrestaShop 9 Alpha?**
+
+You can try upgrading using "Local archive" method in the [autoupgrade](https://github.com/PrestaShop/autoupgrade). However, please remember that this version is not stable and should not be used in production environments. We haven't tested the upgrade path yet, so it may not work as expected.
```

Comment:
```suggestion
You can try upgrading using the "Local archive" method in the [autoupgrade](https://github.com/PrestaShop/autoupgrade). However, please remember that this version is not stable and should not be used in production environments. We haven't tested the upgrade path yet, so it may not work as expected.
```